src/core/public/http/fetch.ts

```diff
@@ -123,6 +123,7 @@ export class Fetch {
         'Content-Type': 'application/json',
         ...options.headers,
         'kbn-version': this.params.kibanaVersion,
+        ...options.context?.toHeader(),
```

Comment:
We will remove this dependency when the RUM agent implements https://github.com/elastic/apm-agent-rum-js/issues/1040. It will take care of passing `context` through the network.